src/components/Gradebook/footer.scss

```diff
@@ -0,0 +1,168 @@
+.max-width-222 {
+  max-width: 222px;
+}
+
+.max-width-264 {
+  max-width: 264px;
+}
+
+.max-width-1180 {
+  max-width: 1180px;
+}
+
+.max-height-39 {
+  max-height: 39px;
+}
+
+.d-grid {
+  display: grid;
+}
+
+$border-1: 1px solid #e7e7e7;
+
+footer {
+  background-color: #fcfcfc;
+
+  ul {
```

Comment:
Styles should be applied to classes instead of elements.

Reply:
Moved to jsx.